src/infra/main/template.yaml

```diff
@@ -29,6 +29,10 @@ Parameters:
     Description: The ARN of the permissions boundary to apply when creating IAM roles
     Type: String
     Default: none
+  CSLSSubscriptionEndpointArn:
+    Type: String
+    Description: CSLS Subscription endpoint
+    Default: "arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython"
```

Comment:
Hey, seems that we will now use this destination instead for any new accounts we will onboard, so new destination is >
`arn:aws:logs:eu-west-2:885513274347:destination:csls_cw_logs_destination_prodpython-2 `as they had some scaling limitations with CSLS, so If you could change that over please - Thank you!